src/styles/blocks/_calendar.scss

```
@@ -0,0 +1,66 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  align-content: center;
+  padding: 10px;
+  gap: 1px;
+
+  $days: 
+    "mon" 0,
+    "tue" $square-side + $indentations-size,
+    "wed" 2 * ($square-side + $indentations-size),
+    "thu" 3 * ($square-side + $indentations-size),
+    "fri" 4 * ($square-side + $indentations-size),
+    "sat" 5 * ($square-side + $indentations-size),
+    "sun" 6 * ($square-side + $indentations-size);
+
+  @each $start-day, $left-indentation in $days {
+    &--start-day--#{$start-day} 
+    &__day:first-child {
+      margin-left: $left-indentation;
+    }
+  }
+
+  @for $days-in-month from 28 through 32 {
+    &--month-length--#{$days-in-month}
+    &__day:nth-child(n + #{$days-in-month + 1}) {
+      display: none;
+    }
```

I'm not sure as to why you wrote it this way, you could try something like this

```suggestion
    &--month-length--#{$days-in-month} :nth-child(n + #{$days-in-month + 1}) {
      display: none;
    }
```

---

I wrote this because of a linter error (string max-size)

---

he just specified that it will only work for an `nth-child` with the `calendar__day` class. not necessarily a mistake